indexer/s3_bucket.tf

```diff
@@ -22,19 +22,34 @@ resource "aws_s3_bucket" "indexer_full_node_snapshots" {
   }
 }
 
+# Enable S3 snapshot lifecycle to clean up old snapshots
+resource "aws_s3_bucket_lifecycle_configuration" "indexer_full_node_snapshots" {
+  count  = var.enable_s3_snapshot_lifecycle ? 1 : 0
+  bucket = aws_s3_bucket.indexer_full_node_snapshots.id
```

Comment:
are there other buckets (i.e. load_balancer logs) that the lifecycle should apply to?

Reply:
Good idea. I will add lifecycle to other buckets